### PR TITLE
Add new table to paraswap Base trades

### PR DESCRIPTION
**Pull request by PablloZz**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:**
Add a `SwappedDirect` table to Paraswap Base trades to include missed transactions.
Examples: [1](https://basescan.org/tx/0x98d900c692582445f2d327a4a7dd7c3050c05d8148af38a0ac24721ccc52ba81), [2](https://basescan.org/tx/0xd1a6b3161fad8effde6280fd966fe606aea89692d001998291995080d6ab5d66), [3](https://basescan.org/tx/0x7be5df2195108dba496ecdebcd21b97646b2a2b0b14180b7e106662361023b2e)

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by PablloZz** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MaksHladun** [comment]
I have read the CLA Document and I hereby sign the CLA

File: dbt_subprojects/dex/models/_projects/paraswap/base/paraswap_v5_base_trades.sql [hunk 1]
```diff
@@ -14,6 +14,7 @@
 {% set trade_event_tables = [
     source('paraswap_base', 'AugustusSwapper_evt_BoughtV3')
     ,source('paraswap_base', 'AugustusSwapper_evt_SwappedV3')
+    ,source('paraswap_base', 'AugustusSwapper_evt_SwappedDirect')
```
Comment (Hosuke) [inline dbt_subprojects/dex/models/_projects/paraswap/base/paraswap_v5_base_trades.sql:17]: Please add the following source schema in `sources/paraswap/base/paraswap_base_sources.yml`:
```
      - name: AugustusSwapper_evt_SwappedDirect
        loaded_at_field: evt_block_time
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @PablloZz

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**